engines/scumm/script_v6.cpp

```diff
@@ -495,6 +495,80 @@ void ScummEngine_v6::o6_pushByteVar() {
 }
 
 void ScummEngine_v6::o6_pushWordVar() {
+	#if defined(USE_ENET) && defined(USE_LIBCURL)
```

Comment:
Could you indent `#if defined(USE_ENET) && defined(USE_LIBCURL)` so there is no tab/space before it?